tensorflow/compiler/tests/matrix_triangular_solve_op_test.py

```diff
@@ -87,6 +87,13 @@ def testBasicNotActuallyTriangular(self):
       self._VerifyTriangularSolveCombo(a.astype(dtype), b.astype(dtype))
 
   def testBasicComplexDtypes(self):
+
+    if xla_test.test.is_built_with_rocm():
+      # TODO(rocm)
```

Comment:
The TF convention is to have usernames of responsible people inside the TODO block, not a feature name. It might be better to remove this comment TODO entirely

Reply:
@cheshire 

removed the comment, and also fixed the linter errors. 

please re-approve. thanks